<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Comparison Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_upload_image2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 2</string>
    </property>
   </widget>
   <!-- 检材图像显示 -->
   <widget class="QLabel" name="label_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 样本图像显示 -->
   <widget class="QLabel" name="label_image2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 检验说明输入框 -->
   <widget class="QTextEdit" name="textedit_inspection_notes">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Add inspection notes...</string>
    </property>
   </widget>
   <!-- 保存检验报告按钮 -->
   <widget class="QPushButton" name="button_save_report">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Report</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
